File: python/7 kyu/61.py
# https://www.codewars.com/kata/5603a4dd3d96ef798f000068/train/python

# Passed

def share_price(invested, changes):
    for change in changes:
        invested *= (100 + change)/100

    invested = str(float(round(invested, 2)))
    if len(invested.split(".")[-1]) != 2:
        invested += '0'

    return invested

output = share_price(1000, [0, 2, 3, 6])
print(output)

File: nodejs/7 kyu/61.js
// https://www.codewars.com/kata/584dc1b7766c2bb158000226/train/javascript

// Passed

function choreAssignment(chores) {
  // sorting the chores
  chores.sort((a, b) => a - b);

  // The maximal workload would be max(chores) + min(chores)
  let workloads = [];
  for (let index = 0; index < chores.length / 2; index++) {
    const workload = chores[index] + chores[chores.length - index - 1];
    workloads.push(workload);
  }

  // sorting workloads
  workloads.sort((a, b) => a - b);

  return workloads;
}

const output = choreAssignment([1, 5, 2, 8, 4, 9, 6, 4, 2, 2, 2, 9]);
console.log(output);
